### DISCUSSION TITLE
NotServedByPagesError Help

### DISCUSSION THREAD

**Discussion by najanijohnson** [community/community#129926] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hey all,
I’m receiving a not served by pages error despite following all the instructions outlined. My GitHub page redirects to the custom domain, but it doesn’t upload my shiny app for some reason. I’m not really sure why. Can anyone help me?

**Comment by LiteBrite82** [comment]
Hi @najanijohnson 
can you share the repo link? CNAME / DNS info?
